packages/flutter/test/analysis_options.yaml

```diff
@@ -0,0 +1,7 @@
+include: ../analysis_options.yaml
+
+linter:
+  rules:
+    # Turn off for tests, since we often use dynamic as a method to obtain a
+    # reference to a private state object or renderer in tests.
+    avoid_dynamic_calls: true
```

Comment:
Remove the comment above as we are no longer turning this off for tests?

Reply:
Oh, yeah, I guess so.  This entire file is probably going away once I've merged the PR that turns it on for the rest of the repo, so it's not terribly important.